src/pages/api/auth-connect/send-identity-validation-code.ts

```diff
@@ -33,84 +33,87 @@ import { getIdentityType } from "@src/utils/get-identity-type";
 const tracer = getTracer();
 
 const handler: Handler = (request, response): Promise<void> => {
-  return tracer.span("send-identity-validation-code handler", async (span) => {
-    const { callbackUrl, identityInput, identityToUpdateId } = request.body;
-
-    const userCookie = await getServerSideCookies<UserCookie>(request, {
-      cookieName: "user-cookie",
-      isCookieSealed: true,
-      cookieSalt: config.cookieSalt,
-    });
-
-    if (userCookie) {
-      const identity = {
-        type: getIdentityType(identityInput.type),
-        value: identityInput.value,
-      };
-
-      return await sendIdentityValidationCode(config.managementCredentials, {
-        callbackUrl,
-        identity,
-        localeCodeOverride: "en-EN",
-        userId: userCookie.sub,
-      })
-        .then(async ({ eventId }) => {
-          span.setDisclosedAttribute("is validation code sent", true);
-
-          let temporaryIdentity: TemporaryIdentity = {
-            eventId: eventId,
-            value: identityInput.value,
-            type: identityInput.type,
-            expiresAt: identityInput.expiresAt,
-            primary: identityInput.primary,
-          };
-
-          if (identityToUpdateId) {
-            temporaryIdentity = {
-              ...temporaryIdentity,
-              identityToUpdateId,
+  return tracer.span(
+    "re-send-identity-validation-code handler",
```

Comment:
I think you confused both files for the span name 🙂

Reply:
https://i.giphy.com/media/XsUtdIeJ0MWMo/200.webp